### DISCUSSION TITLE
求助！！！无法登录！！！

### DISCUSSION THREAD

**Discussion by ClariS1021** [Richasy/Bili.Uwp#1761] [问答]
使用密码登录失败，转为二维码登录，手机客户端提示哔哩哔哩版本过低，无法登录。
UWP版本 2.2307.1.0 X64
手机客户端版本 7.44
系统版本 Win10 22H2
[S30829-23424799.png](https://github.com/Richasy/Bili.Uwp/assets/143527954/9630db89-71cb-47f5-9ffb-e21b5bba75b6)

**Comment by Xmug** [comment]
我也是这个问题，现在我改用作者的另一个应用哔哩助理了

**Reply by 2322131090** [reply]
你用的那个，我也是无法登录
